cypress/fixtures/commonComponents/alert.json

```diff
@@ -3,22 +3,28 @@
     
   },
   "titleOtherLanguage": {
-    "title": "mp150ú¿¡üßä"
+    "title": "",
+    "titleSpecialCharacters": "otherLanguage"
```

Comment:
Is it right that `otherLanguage` is wrapped in `""` here? Same question with all of these

Reply:
yes, it is an parameter passed as a `string`